hedera-node/src/main/java/com/hedera/services/ledger/AccountsCommitInterceptor.java

```diff
@@ -0,0 +1,67 @@
+package com.hedera.services.ledger;
+
+import com.hedera.services.context.SideEffectsTracker;
+import com.hedera.services.ledger.properties.AccountProperty;
+import com.hedera.services.state.merkle.MerkleAccount;
+import com.hederahashgraph.api.proto.java.AccountID;
+
+import java.util.List;
+import java.util.Map;
+
+public class AccountsCommitInterceptor implements
+		CommitInterceptor<AccountID, MerkleAccount, AccountProperty> {
+
+	// The tracker this interceptor should use for previewing changes. The interceptor is NOT
+	// responsible for calling reset() on the tracker, as that will be done by the client code.
+	private SideEffectsTracker sideEffectsTracker;
```

Comment:
While I like what you did to make the interceptor optional on `TransactionalLedger`, here it does seem preferable to use constructor injection for the `sideEffectsTracker`. (Since the interceptor is not usable unless it has a non-null `sideEffectsTracker`.)